.circleci/config.yml

```diff
@@ -152,6 +153,43 @@ commands:
 
             conbench cpp-micro --run-name="$run_name" --run-reason="$run_reason"
 
+  fuzzer-runs:
+    parameters:
+      fuzzer_output:
+        type: string
+        default: "/tmp/fuzzer.log"
```

Comment:
Do we need these defaults? It is confusing that some defaults apply to Aggregate Fuzzer, but some to Join Fuzzer.

Reply:
Agree, defaults should either be general enough to apply to all or not be set imo. (or be consistenly set for one type of fuzzer but not a fan)